### PR TITLE
[#2833] unzip resource files to a folder

### PR DESCRIPTION
**Pull request by sblack-usu**
### Pull Request Checklist: 
- [ ] Positive Test Case Written by Dev

- [ ] Automated Testing

- [ ] Sufficient User and Developer Documentation

- [ ] Passing Jenkins Build

- [ ] Peer Code review and approval

### Positive Test Case
1. [Enter positive test case here]

### CODE REVIEW

File: hs_core/views/utils.py [hunk 1]
```diff
@@ -680,9 +680,8 @@ def unzip_file(user, res_id, zip_with_rel_path, bool_remove_original):
     if not resource.supports_unzip(zip_with_rel_path):
         raise ValidationError("Unzipping of this file is not supported.")
 
-    unzip_path = os.path.dirname(zip_with_full_path)
     zip_fname = os.path.basename(zip_with_rel_path)
-    istorage.session.run("ibun", None, '-xDzip', zip_with_full_path, unzip_path)
+    unzip_path = istorage.unzip(zip_with_full_path)
     link_irods_folder_to_django(resource, istorage, unzip_path, (zip_fname,))
```
[the remark this answers is not in the document]
Comment (sblack-usu) [reply hs_core/views/utils.py]: nice catch, I've removed it.

File: hs_composite_resource/tests/test_composite_resource.py [hunk 2]
```diff
@@ -1289,3 +1292,58 @@ def test_supports_delete_original_folder_on_zip(self):
...
+        zip_file_rel_path = os.path.join('data', 'contents', zip_res_file.file_name)
+        unzip_file(self.user, self.composite_resource.short_id, zip_file_rel_path,
+                   bool_remove_original=False)
+
+        # resource should have 3 files now
+        self.assertEqual(self.composite_resource.files.count(), 3)
+        # there should not be any resource files ending with _meta.xml or _resmap.xml
+        for res_file in self.composite_resource.files.all():
+            self.assertFalse(res_file.file_name.endswith('_mata.xml'))
+        self.assertFalse(res_file.file_name.endswith('_resmap.xml'))
```
Comment (pkdash) [inline hs_composite_resource/tests/test_composite_resource.py]: I think it needs to be inside the ```for``` loop

Comment (sblack-usu) [reply hs_composite_resource/tests/test_composite_resource.py]: fixed

**Comment by sblack-usu** [comment]
yeah, it'll go in the composite resource phase 2 first... it's a solution to a bug that we found when unzipping at the aggregation level.

**Comment by sblack-usu** [comment]
@pkdash - Is this good to merge into the composite resource branch now?

**Pull request merged**